README.md

```diff
@@ -27,22 +30,24 @@ pip install .
 
 ## Using the posterior database from python
 
-The included database contains convenience functions to access data, model code and information for individual posteriors.
+The included database contains convenience functions to access data, model code, and information for individual posteriors. We can create this database with either local or online access to `posteriordb`.
```

Comment:
Ideally these things would not be written in the third person, not in second person singular (you/you/yours) or plural (we/us/ours).

Reply:
Also fixed.